torchvision/utils.py

```diff
@@ -382,6 +389,116 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
+# Flow visualization code adapted from https://github.com/tomrunia/OpticalFlow_Visualization
+@torch.no_grad()
+def flow_to_image(
+    flow: torch.Tensor,
+) -> torch.Tensor:
+
+    """
+    Converts the two-dimensional flow to an RGB Image.
+
+    Args:
+        flow (Tensor): Flow of shape (2, H, W) and dtype torch.float.
+
+    Returns:
+        img (Tensor(3, H, W)): Image Tensor where each color corresponds to a given flow direction.
+    """
+
+    if flow.ndim != 3 or flow.size(0) != 2:
+        raise ValueError(f"Input flow should have shape (2, H, W), got {flow.shape}.")
+
+    max_norm = torch.sum(flow ** 2, dim=0).sqrt().max()
+    epsilon = torch.finfo((flow).dtype).eps
+    normalized_flow = flow / (max_norm + epsilon)
+    return _normalized_flow_to_colors(normalized_flow)
+
+
+@torch.no_grad()
+def _normalized_flow_to_colors(normalized_flow: torch.Tensor) -> torch.Tensor:
```

Comment:
It's a private function so I won't insist more, but I think it still makes sense to name this `_normalized_flow_to_image` because:

- this accurately describes what this function returns: it returns a tensor image, not "colors"
- we already have`flow_to_image()` which returns the same thing as this function.